lib/image/index.js

```diff
@@ -53,12 +73,58 @@ module.exports = class Image {
         );
     }
 
+    frame(area, opts = {}) {
+        area = this._scale(area, opts.scaleFactor);
```

Comment:
why you scale ignore area here again? First you ignored it inside `setIgnoreAreas`

Reply:
Because this area is not scaled. 
Here scaled areas are being bound to the image (to be used later by looks-same)
https://github.com/gemini-testing/gemini-core/pull/61/files#diff-de720270c5a44915a962b3bd05959840R33

Notice that `_scale` returns new object
https://github.com/gemini-testing/gemini-core/pull/61/files#diff-3d258dfeed9f2b14efd5934190f83c42L65-L75